.circleci/config.yml

```diff
@@ -28,3 +28,6 @@ jobs:
     - persist_to_workspace:
         root: ~/go/src/github.com/Clever
         paths: "."
+    - run: if [ "${CIRCLE_BRANCH}" == "master" ]; then cat ./swagger.yml | grep "^  version:" | cut -d":" -f2 | tr -d " " > ./VERSION; fi;
+    - run: if [ "${CIRCLE_BRANCH}" == "master" ]; then $HOME/ci-scripts/circleci/github-release $GH_RELEASE_TOKEN; fi;
```

Comment:
I checked https://app.circleci.com/settings/project/github/Clever/discovery-go/environment-variables?return-to=https%3A%2F%2Fapp.circleci.com%2Fpipelines%2Fgithub%2FClever%2Fdiscovery-go%3Ffilter%3Dall doesn't have GH_RELEASE_TOKEN defined in it's env, probably because this is an old repo. You will have to add that manually for this to work. As this is a public repo, you should copy the release token from `ark secrets read production.init-service github-release-token-public` and add it in circle ci.

Reply:
Added, and thank you!